app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = current_user.comments.new(comment_params)
+    @post = Post.find(params[:comment][:post_id])
+
+    if @comment.save
+      redirect_to "#{users_path}/#{@post.author_id}/posts/#{@post.id}"
```

Comment:
- [ ] [OPTIONAL] You've implemented this method really well. remember that you can use a URL helpers that "Rails" provides to navigate directly to make this line of code more succinct.
```suggestion
      redirect_to user_post_path @post
```